com.unity.cinemachine/Runtime/Components/CinemachineSplineDolly.cs

```diff
@@ -21,30 +21,10 @@ namespace Unity.Cinemachine
     [HelpURL(Documentation.BaseURL + "manual/CinemachineSplineDolly.html")]
     public class CinemachineSplineDolly : CinemachineComponentBase
     {
-        /// <summary>The Spline container to which the camera will be constrained.</summary>
-        [Tooltip("The Spline container to which the camera will be constrained.")]
-        public SplineContainer Spline;
-
-        /// <summary>The position along the spline at which the camera will be placed. This can be animated directly,
-        /// or set automatically by the Auto-Dolly feature to get as close as possible to the Follow target.
-        /// The value is interpreted according to the Position Units setting.</summary>
-        [Tooltip("The position along the spline at which the camera will be placed.  "
-            + "This can be animated directly, or set automatically by the Auto-Dolly feature to "
-            + "get as close as possible to the Follow target.  The value is interpreted "
-            + "according to the Position Units setting.")]
-        public float CameraPosition;
-
-        /// <summary>How to interpret the Spline Position:
-        /// - Distance: Values range from 0 (start of Spline) to Length of the Spline (end of Spline).
-        /// - Normalized: Values range from 0 (start of Spline) to 1 (end of Spline).
-        /// - Knot: Values are defined by knot indices and a fractional value representing the normalized
-        /// interpolation between the specific knot index and the next knot."</summary>
-        [Tooltip("How to interpret the Spline Position:\n"
-            + "- Distance: Values range from 0 (start of Spline) to Length of the Spline (end of Spline).\n"
-            + "- Normalized: Values range from 0 (start of Spline) to 1 (end of Spline).\n"
-            + "- Knot: Values are defined by knot indices and a fractional value representing the normalized " 
-            + "interpolation between the specific knot index and the next knot.\n")]
-        public PathIndexUnit PositionUnits = PathIndexUnit.Normalized;
```

Comment:
What happen's with serialization? It breaks the running sample scene.

Reply:
Are you checking with freshly-installed samples?  I'm supposed to have upgraded the running race scene